styles.css

```diff
@@ -482,3 +492,285 @@ textarea {
+    align-items: center;
+  }
+
+  .vector {
+    height: 1px;
+    background-color: #212121;
+    margin-right: 10%;
+    margin-left: 0;
+  }
+
+  .title-works {
+    padding: 135px 0;
+  }
+
+  .stories-section {
+    display: grid;
+    grid-template-columns: 1.5fr 1fr;
+    column-gap: 24px;
+  }
+
+  .stories-section img {
+    width: 100%;
+    margin: 0;
+  }
+
+  .stories-section section {
+    margin-right: 10%;
+  }
+
+  .projects {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 37px 24px;
+  }
+
+  .languages ul {
+    padding: 0 25%;
+    align-items: flex-start;
+  }
+
+  .projects-container h2,
+  .projects-container p {
+    margin: 16px 16px;
+  }
+
+  .projects-container:first-child {
+    background-image: url("img/imgplaceholder1.png");
+    background-size: cover;
+    padding: 0;
+    display: flex;
+    flex-direction: column-reverse;
+    border: solid 1px black;
+  }
+
+  .projects-container:first-child h2,
+  .projects-container:first-child p,
+  .projects-container:first-child ul {
+    display: none;
+  }
+
+  .projects-container:nth-child(2n) {
+    background-image: url("img/imgplaceholder2.png");
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+
+  .projects-container:nth-child(3n) {
+    background-image: url("img/imgplaceholder3.png");
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+
+  .projects-container:nth-child(4n) {
+    background-image: url("img/imgplaceholder4.png");
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+
+  .projects-container:nth-child(5n) {
+    background-image: url("img/imgplaceholder2.png");
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+
+  .projects-container:last-child {
+    background-image: url("img/imgplaceholder3.png");
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+
+  .projects-container:nth-child(2n) a,
+  .projects-container:nth-child(3n) a,
+  .projects-container:nth-child(4n) a,
+  .projects-container:nth-child(5n) a,
+  .projects-container:last-child a {
+    display: none;
+  }
+
+  .container {
+    max-width: 85%;
+    margin: 20px auto;
+  }
+
+  .techs-container {
+    margin: 20px 0 32px 0;
+  }
+
+  .button-project {
+    margin: 0;
+    width: 100%;
+  }
+
+  .multi {
+    margin: 0;
+  }
+
+  .about {
+    background-image: none;
+    display: flex;
+    justify-content: center;
+    padding-bottom: 130px;
+    align-items: center;
+    column-gap: 40px;
+  }
+
+  .about section {
+    margin-left: 10%;
+    text-align: justify;
+    font-size: 20px;
+  }
+
+  .about img {
+    width: 50%;
+    height: 50%;
+    margin-top: 100px;
+  }
+
+  .vector-about {
+    margin-top: 0;
+  }
```

Comment:
- [ ] If you check the original design, this vector does not take the whole width of the page, like in your design. You should consider making this a bit smaller and then center it.

Figma design:
![image](https://user-images.githubusercontent.com/66967274/144541494-aa3ec0a4-bde9-4091-a2b0-81a37e3a4665.png)

Your design:
![image](https://user-images.githubusercontent.com/66967274/144541571-a48d5f4b-d391-4eda-8b2a-504d357fd2aa.png)